AimsUI/LayerManager.py

```diff
@@ -339,10 +340,30 @@ def installRefLayers(self):
         Install AIMS postgres reference layers
         """
         
-        refLayers ={'par':( 'par', 'lds', 'primary_parcels', 'id', True, "ST_GeometryType(shape) in ('ST_MultiPolygon', 'ST_Polygon' )",'Parcels' ) ,
-                    'lpr':( 'lpr', 'lds', 'primary_parcels', 'id', True, "ST_GeometryType(shape) in ('ST_MultiPolygon', 'ST_Polygon' )",'Parcels (Labels)' ) ,
-                    'rcl':( 'rcl', 'roads', 'simple_road_name_view', 'uid', True, "",'Roads' ),
-                    'ppr':( 'ppr', 'lds', 'all_parcels_pend', 'id', True, "ST_GeometryType(shape) in ('ST_MultiPolygon', 'ST_Polygon' )",'Pending Parcels' )
+        '''
+        
+        in dev - optimisation needs to be explored
+        
+        sql = """(SELECT ROW_NUMBER() OVER (ORDER BY p.id ASC) AS uid, p.id, a.appellation_value, p.shape FROM bde.crs_parcel p
```

Comment:
@SPlanzer: This query won't provide the complete application string. e.g a.appellation_value is only one part of the full appellation string. The function bde.bde_get_combined_appellation(p_par_id INTEGER, p_long CHAR) can do that, however it's very slow to run in real-time. What's the max scale that you need the parcels layer to show up?